src/tests/Microsoft.Diagnostics.Monitoring.EventPipe/AspNetTriggerUnitTests.cs

```diff
@@ -196,6 +197,20 @@ public void TestAspNetStatus()
             ValidateTriggers(trigger, s1, s2, s3, e3, e1, e2, s4, s5, s6, s7, e5, e4, e6, e7);
         }
 
+        [Fact]
+        public void TestInvalidPaths()
+        {
+            AspNetRequestCountTriggerSettings settings = new()
+            {
+                IncludePaths = new[] { "/SomePath/***/*" },
```

Comment:
Should we also check for Regex location tokens like `^` and `$`? I think if there are multiple of those we can incur an `ArgumentFormatException` from `Regex.ctor`. Are we allowing users to put in arbitrary regex in the paths? Now that I think about it, we may want to restrict user input to "valid URL characters" + `*` and `/**`.

Reply:
As mentioned before, we always escape the input first. If the user puts in `^^someUrl`, we will search for the literal `^^someUrl` (`\^\^someUrl`as a regex). 

We effectively honor * and **. I avoided the url validation because even though there's a url spec, it's not always followed making it challenging to validate.